### PR TITLE
Add ako skip delete annotation to clusterbootstrap

### PR DESCRIPTION
**Pull request by chenlin07**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Add ako skip delete annotation to clusterbootstrap

Fixes #

**Describe testing done for PR**:

**Special notes for your reviewer**:

**Release note**:

```release-note

```
**New PR Checklist**

- [ ] Ensure PR contains only public links or terms
- [ ] Use good commit [messages](https://github.com/vmware-tanzu/tanzu-framework/blob/main/CONTRIBUTING.md)
- [ ] Squash the commits in this branch before merge to preserve our git history
- [ ] If this PR is just an idea or POC, use a [Draft PR](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests#draft-pull-requests) instead of a full PR
- [ ] Add appropriate [labels](https://github.com/vmware-tanzu/load-balancer-operator-for-kubernetes/labels) according to what type of issue is being addressed.

### CODE REVIEW

File: controllers/akodeploymentconfig/cluster/cluster_controller_addon_secret.go [hunk 1]
```diff
@@ -250,6 +250,18 @@ func (r *ClusterReconciler) patchAkoPackageRefToClusterBootstrap(ctx context.Con
...
 
+	// Add skip deleting ako packageinstall annotation to clusterboostrap if it doesn't exist
+	if _, exist := bootstrap.Annotations[akoov1alpha1.TKGSkipDeletePkgiAnnotationKey]; !exist {
+		if bootstrap.Annotations == nil {
+			bootstrap.Annotations = make(map[string]string)
+		}
+		bootstrap.Annotations[akoov1alpha1.TKGSkipDeletePkgiAnnotationKey] += "load-balancer-and-ingress-service, "
+	} else {
+		if !strings.Contains(bootstrap.Annotations[akoov1alpha1.TKGSkipDeletePkgiAnnotationKey], "load-balancer-and-ingress-service") {
+			bootstrap.Annotations[akoov1alpha1.TKGSkipDeletePkgiAnnotationKey] += "load-balancer-and-ingress-service, "
```
Comment (lubronzhan) [inline controllers/akodeploymentconfig/cluster/cluster_controller_addon_secret.go:261]: bootstrap.Annotations[akoov1alpha1.TKGSkipDeletePkgiAnnotationKey] += ",load-balancer-and-ingress-service"
The comma should be at the front

Comment (chenlin07) [reply controllers/akodeploymentconfig/cluster/cluster_controller_addon_secret.go]: Thanks! updated

File: controllers/akodeploymentconfig/cluster/cluster_controller_addon_secret.go [hunk 2]
```diff
@@ -275,6 +287,11 @@ func (r *ClusterReconciler) removeAkoPackageRefFromClusterBootstrap(ctx context.
 		}
 	}
 
+	// Remove skip deleting ako packageinstall annotation to clusterboostrap if it exists
```
Comment (XudongLiuHarold) [inline controllers/akodeploymentconfig/cluster/cluster_controller_addon_secret.go:290]: This is not right, we should remove the annotation when avi resource clean up finished

File: controllers/akodeploymentconfig/cluster/cluster_controller_addon_secret.go [hunk 3]
```diff
@@ -250,6 +250,18 @@ func (r *ClusterReconciler) patchAkoPackageRefToClusterBootstrap(ctx context.Con
 		}
 	}
 
+	// Add skip deleting ako packageinstall annotation to clusterboostrap if it doesn't exist
+	if _, exist := bootstrap.Annotations[akoov1alpha1.TKGSkipDeletePkgiAnnotationKey]; !exist {
+		if bootstrap.Annotations == nil {
+			bootstrap.Annotations = make(map[string]string)
+		}
+		bootstrap.Annotations[akoov1alpha1.TKGSkipDeletePkgiAnnotationKey] += ",load-balancer-and-ingress-service"
```
Comment (XudongLiuHarold) [inline controllers/akodeploymentconfig/cluster/cluster_controller_addon_secret.go:258]: replace all `load-balancer-and-ingress-service` string using a const var?

**Review by XudongLiuHarold — APPROVED** [review]

**Pull request merged**